scripts/lint-staged/src/eslint.js

```diff
@@ -16,69 +16,64 @@ const exec = promisify(child_process.exec);
  */
 const eslintForPackageScript = path.join(__dirname, 'eslint-for-package.js');
 
-const files = process.argv.slice(2);
-
 /**
  * Since we have an eslint config file per package we need to respect this when running
  * eslint for staged files. To do this we group the files per package name. This function takes
  * a list of package names and returns an object with the package root as the key and the files
  * in that package as the value.
+ *
+ * @param {string[]} files
  * @returns {{ [packagePath: string]: string[] }}
  */
-function groupFilesByPackage() {
+function groupFilesByPackage(files) {
```

Comment:
files was a global which is not the best pattern - refactored to be passed via func param